### DISCUSSION TITLE
LinX / AiDEX X Support via Companion App

### DISCUSSION THREAD

**Discussion by Stealthwick** [NightscoutFoundation/xDrip#3532] [Ideas]
Hey team, I've been testing the new sensors from MicroTech Medical which are branded AiDEX X in Asia and will be released as LinX in Europe. So far I've been impressed by the data but would link to import to xDrip so that I can do some more robust comparisons.

The sensor reading refreshes every minute. There is a system notification that looks like this:
![Screenshot_20240610_193446_LinX](https://github.com/NightscoutFoundation/xDrip/assets/40660629/331d12b5-8b98-46d3-b7da-e0d8d6d70041)

I would have thought that this would be enough to run in companion mode, but no readings coming through. Any chance we can fix this? 

The LinX app can be found in the Google Play store here: https://play.google.com/store/apps/details?id=com.microtech.aidexx.mgdl

**Comment by Navid200** [comment]
Linx was added to the companion app list in June of 2024.

And these are the instructions for setting up xDrip to use Companion App mode: https://navid200.github.io/xDrip/docs/Follow/CompanionApp.html

**Comment by Ginocar** [comment]
> Hey team, I've been testing the new sensors from MicroTech Medical which are branded AiDEX X in Asia and will be released as LinX in Europe. So far I've been impressed by the data but would link to import to xDrip so that I can do some more robust comparisons.
> 
> The sensor reading refreshes every minute. There is a system notification that looks like this:
> ![Screenshot_20240610_193446_LinX](https://github.com/NightscoutFoundation/xDrip/assets/40660629/331d12b5-8b98-46d3-b7da-e0d8d6d70041)
> 
> I would have thought that this would be enough to run in companion mode, but no readings coming through. Any chance we can fix this? 
> 
> The LinX app can be found in the Google Play store here: https://play.google.com/store/apps/details?id=com.microtech.aidexx.mgdl
> 

Hello,
Did you set it success?
Could you help me?
I tried to set it but I could not.
If I choose the Aidex app in the xdrip source menu I cannot see any VC data in the xdrip.

**Comment by Stealthwick** [comment]
You need to set data source to Companion App mode - not AiDEX

**Comment by Ginocar** [comment]
> You need to set data source to Companion App mode - not AiDEX
> 
> On Sun, 20 Apr 2025, 6:57 pm Ginocar, ***@***.***> wrote:
> 
> > Hey team, I've been testing the new sensors from MicroTech Medical which
> > are branded AiDEX X in Asia and will be released as LinX in Europe. So far
> > I've been impressed by the data but would link to import to xDrip so that I
> > can do some more robust comparisons.
> >
> > The sensor reading refreshes every minute. There is a system notification
> > that looks like this:
> > Screenshot_20240610_193446_LinX.jpg (view on web)
> > <https://github.com/NightscoutFoundation/xDrip/assets/40660629/331d12b5-8b98-46d3-b7da-e0d8d6d70041>
> >
> > I would have thought that this would be enough to run in companion mode,
> > but no readings coming through. Any chance we can fix this?
> >
> > The LinX app can be found in the Google Play store here:
> > https://play.google.com/store/apps/details?id=com.microtech.aidexx.mgdl
> >
> > Hello,
> > Did you set it success?
> > Could you help me?
> > I tried to set it but I could not.
> > If I choose the Aidex app in the xdrip source menu I cannot see any VC
> > data in the xdrip.
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/NightscoutFoundation/xDrip/discussions/3532#discussioncomment-12888630>,
> > or unsubscribe
> > <https://github.com/notifications/unsubscribe-auth/AJWG5FJYX24V23JCDAFZZOT22NAMRAVCNFSM6AAAAABW6DNOKOVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTEOBYHA3DGMA>
> > .
> > You are receiving this because you authored the thread.Message ID:
> > ***@***.***
> > .com>
> >
> 

Thank you so much.
It works

**Comment by joshferns** [comment]
Hi,
I have the Vixxa 1 app, this again is a variant of AiDex X in India.
Request that this be added to the coOptedPackages
Here's the link to the Playstore:

https://play.google.com/store/apps/details?id=com.microtech.aidexx.grx1.mmoll

Also, my app displays units in mgdl though the link says mmol.
Please let me know if any further information is needed from my end.

**Reply by joshferns** [reply]
<img width="1080" height="276" alt="Screenshot_20260803_130529_Claude" src="https://github.com/user-attachments/assets/6c9c5f9c-71c6-4316-b72b-ae72d0613694" />

Adding a screenshot of my notification for reference.

**Reply by joshferns** [reply]
I've opened #4673 with the two-line addition for `com.microtech.aidexx.grx1.mmoll` (added to both `coOptedPackages` and `coOptedPackagesAll`, alongside the other MicroTech variants).

Built it and tested on my own phone with a live sensor — Companion App mode now collects readings from the Vixxa app correctly, with matching values, delta and trend. Screenshot below.
<img width="1080" height="433" alt="Screenshot_20260804_215608_One UI Home" src="https://github.com/user-attachments/assets/778e2780-35ae-44b0-99bc-aa57bb73fabf" />

**Reply by Navid200** [reply]
Isn't that identical to an existing PR?
https://github.com/NightscoutFoundation/xDrip/pull/4665

**Reply by joshferns** [reply]
Yes, its identical, I'll close mine.
Thanks.

**Reply by Navid200** [reply]
@joshferns 
Thanks for your work.

Sorry about that.   I should have made a comment here in this discussion when I opened that PR.    

Please keep up the good work.
